tools/trace_analyzer_tool.cc

```diff
@@ -1428,7 +1429,7 @@ Status TraceAnalyzer::OpenStatsOutputFiles(const std::string& type,
                          &new_stats.a_qps_f);
   }
 
-  return Status::OK();
+  return s;
```

Comment:
That entire method is kind of screwy.  It keeps calling Open over and over again and never checking the Status.  Of course, if the status was not OK, the program will exit.  I am not sure what is best to do here.